docusaurus/docs/reactnative/ui-components/auto_complete_input.mdx

```diff
@@ -26,7 +26,7 @@ Component to render input box within within the [`MessageInput`](./message_input
 This component provides autocomplete functionality for the commands and mentions feature.
 Commands suggestions will show up when user types "`/`", while mentions suggestions will show up when user types "`@`".
 
-Please check the guide of [Customize Message Input](../guides/message_input.mdx) for usage of this component.
+Please check the guide of [Customize Message Input](../guides/message_input_customization.mdx) for usage of this component.
```

Comment:
Check the Customize Message Input guide for more details about how to customize this component